lib/opam_build.ml

```diff
@@ -21,6 +21,11 @@ let opam_install ~variant ~upgrade_opam ~pin ~with_tests ~pkg =
         opam remove -y %s && opam %s%s %s
         res=$?
         test "$res" = 0 && exit 0
+        if test "$res" = 60 && diff -q /usr/bin/opam /usr/bin/opam-2.0; then
+          sudo cp /usr/bin/opam-2.1 /usr/bin/opam
```

Comment:
I believe this should be a hard link and not a cp (as that's what the base dockerfiles do)

Reply:
Fixed.